src/phases/move-phase.ts

```diff
@@ -186,7 +186,7 @@ export class MovePhase extends BattlePhase {
 
     this.lapsePreMoveAndMoveTags();
 
-    if (!(this.failed || this.cancelled)) {
+    if (!this.failed && !this.cancelled) {
```

Comment:
```suggestion
    if (!(this.failed || this.cancelled)) {
```
There's a Biome rule that would do the opposite of what you did (aka make it look like it used to). Why did you change this?

Reply:
that is actually a very good question lol

Reply:
Which rule is that again

Reply:
https://biomejs.dev/linter/rules/use-simplified-logic-expression/
Specifically the fourth example that includes `const r4 = !boolExpr1 || !boolExpr2;`